src/main/java/net/sf/jabref/exporter/layout/format/HTMLParagraphs.java

```diff
@@ -43,7 +43,7 @@ public String format(String fieldText) {
         }
 
         Matcher m = HTMLParagraphs.beforeNewLines.matcher(fieldText);
-        StringBuffer s = new StringBuffer();
+        StringBuffer s = new StringBuffer(19); // At least 19 characters are appended later
```

Comment:
I agree with @tobiasdiez that this optimization is probably more confusing than helping.